sunpy/data/_sample.py

```diff
@@ -104,7 +104,7 @@ def download_sample_data(overwrite=False):
     if not results.errors:
         return results
     else:
-        log.info('Failed to download sample data, retrying with a mirror.')
+        warnings.warn('Failed to download sample data, retrying with a mirror.')
```

Comment:
My instinct here is `info` is correct. I feel like `warning` implies the user can and should do something about it, this is more likely to be a server error for us. We want it to print so we can see and fix those errors.

Reply:
yes sure, updated the PR